newIDE/app/src/SceneEditor/index.js

```diff
@@ -154,6 +154,7 @@ type State = {|
 
   renamedObjectWithContext: ?ObjectWithContext,
   selectedObjectsWithContext: Array<ObjectWithContext>,
+  activeEditor: string | null,
```

Comment:
maybe type this prop with a list of possible editors?

Reply:
Good idea!